shared/teams/team/invite-row/container.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+import * as React from 'react'
+import {connect} from 'react-redux'
+import * as Constants from '../../../constants/teams'
+import * as I from 'immutable'
+import {TeamInviteRow} from '.'
+
+import type {TypedState} from '../../../constants/reducer'
+
+type OwnProps = {
+  username: string,
+  teamname: string,
+}
+
+const getFollowing = (state, username: string) => {
+  const followingMap = Constants.getFollowingMap(state)
+  return !!followingMap[username]
+}
+
+type StateProps = {
+  following: boolean,
+  you: ?string,
+  _invites: I.Set<Constants.InviteInfo>,
+  _members: I.Set<Constants.MemberInfo>,
+}
+
+const mapStateToProps = (state: TypedState, {teamname, username}: OwnProps): StateProps => ({
+  _invites: state.entities.getIn(['teams', 'teamNameToInvites', teamname]),
+  _members: state.entities.getIn(['teams', 'teamNameToMembers', teamname]),
+  following: getFollowing(state, username),
+  you: state.config.username,
+})
+
+type DispatchProps = {}
+
+const mergeProps = (stateProps: StateProps, dispatchProps: DispatchProps, ownProps: OwnProps) => {
+  const user =
```

Comment:
this kinda makes the Set not that valuable if you have to iterate over it like this. fine short term but if you had 2 maps it'd be a quick lookup